test/shared/src/main/scala/zio/test/environment/package.scala

```diff
@@ -382,7 +382,7 @@ package object environment extends PlatformSpecific {
       private lazy val freeze: IO[Unit, Set[Fiber.Status]] =
         supervisedFibers.flatMap { fibers =>
           ZIO
-            .foreach(fibers)(_.status.filterOrFail {
+            .foreach[Any, Unit, Fiber.Runtime[Any, Any], Fiber.Status, Iterable](fibers)(_.status.filterOrFail {
```

Comment:
😱  If there's a way to fix this, we really should explore that.

Reply:
This one is particular bad because `Set` is invariant. All we really need here is a way to widen `Set` to `Iterable`, which unfortunately the compiler won't do for us automatically here. On Scala 2.12 we can just do `foreach(fibers.toIterable)(???)` but that doesn't work on Scala 2.13 because now `toIterable` returns `this.type` so it doesn't actually widen. Let me see if we can use a type ascription on `fibers`.

Reply:
This one is fixed now with the new `foreach` variant for `Set`.